dns_sd_windows.c

```diff
@@ -448,6 +456,16 @@ int dnssd_find_hosts(struct dns_sd_discovery_data **ddata)
 				FD_SET(sockets[isock], &readfs);
 			}
 		}
+
+		/* res > 0 even if we didn't process anything :(
+		 * timeout from the last time we successfully added a proper mdns record
+		 */
+		nowtime = iio_read_counter_us();
+
+		/* convert to ms */
+		diff = (nowtime - lasttime) / 1000LL;
```

Comment:
nit: is the `LL` really needed? In theory, this could actually get wrong, if `lastime > nowtime` but I guess that should be impossible :thinking:

Reply:
Should probably be `ULL` yes ;) Not strictly required but I like to avoid implicit type casting when I can.

Both Windows and POSIX implementations of `iio_read_counter_us` use monotonic clocks, which mean that they can never go backwards. And therefore `lasttime >= nowtime` is always true.